src/ansys/dpf/post/result_workflows/_build_workflow.py

```diff
@@ -96,7 +96,8 @@ def _requires_manual_averaging(
     base_name: str,
     location: str,
     category: ResultCategory,
```

Comment:
This was called with selection = None in the _build_selection, which meant the skin and external_layer did not have an effect.